lib/features/auto-place/AutoPlaceUtil.js

```diff
@@ -109,133 +113,124 @@ export function getConnectedAtPosition(source, position, element) {
 }
 
 /**
-* Compute best distance between source and target,
-* based on existing connections to and from source.
+* Compute optimal distance between source and target based on existing connections to and from source.
 *
 * @param {djs.model.Shape} source
-* @param {string} axis
-* @param {Function} [filter]
 * @param {Object} [hints]
-* @param {djs.model.Shape} [hints.connectionTarget]
-* @param {djs.model.Shape} [hints.maxDistance]
+* @param {number} [hints.defaultDistance]
+* @param {string} [hints.direction]
+* @param {Function} [hints.filter]
+* @param {Function} [hints.getWeight]
+* @param {number} [hints.maxDistance]
+* @param {string} [hints.reference]
 *
-* @return {number} distance
+* @return {number}
 */
-export function getConnectedDistance(source, axis, filter, hints) {
-  if (!filter) {
-    filter = noneFilter;
-  }
-
-  if (isObject(filter)) {
-    hints = filter;
-    filter = noneFilter;
-  }
-
+export function getConnectedDistance(source, hints) {
```

Comment:
This breaking change is not documented in the commits.

Reply:
Thanks for the hint. I've added a comment to the respective commit and to the PR description.

Reply:
Schön!